rust/feature-flags/src/config.rs

```diff
@@ -115,7 +115,7 @@ pub struct Config {
     #[envconfig(default = "1000")]
     pub max_concurrency: usize,
 
-    #[envconfig(default = "10")]
+    #[envconfig(default = "50")]
```

Comment:
our production DB `max_connections` setting is 1000, so we have plenty of headroom and I'm not trying to get close to that yet.  Mostly trying to see if turning this knob helps at all.  I don't _think_ this is infra related since I'm not actually seeing spikes in CPU/Memory during these request spikes, so I'm pretty sure it's application-related.